docs/setup-guide/cluster_setup.md

```diff
@@ -0,0 +1,433 @@
+#	Edge Cluster Multi-Layer Setup and Configuration
+
+## Abstract
+The purpose of this document is how to setup and configuration Cloud core and Edge core , and describe the each step to create virtual machine, setup the port number, install kubernets, GoLang, and so on. Running Cloud core and Edge core and deployed mission and task to Edge node. Improve the Edge computing. This Cloud and Edge design is derived from cloud end to edge end for Edge System Functional Description and the Setup Requirements Specification. The intended user of this program is the edge computing user. 
+1. Virtual Machine Setup (create Cloud core  and Edge core virutal machine, and setup port),
+2. Fornax Installation and Configuration (Install all the kubernetes compnents: kubectl, kubadm, bubelet),
+3. GoLang Installlation and Configuration (Install all GoLang component and load the Fornax source code), 
+4. Generate Machine Security certification, and deployed to virtual machine, 
+5. Install CRD file in Cloud core. 
+6. Run cloud-core and edge-core and deployed mission and verify the mission.
+7. You should be a root user.
+
+## 1.1. Virtual Machine Setup and Configuration (We use AWS for example)
+-	Ubuntu 18.04, Create A virtual machine as an cloud-core, Create B and C virtual machine as edge-core.
+-	Open the port of 10000 and 10002 in the security group of the cloud-core machine and edge-core machine
+- 	Go to doc and follow up instruction to setup: [Virtual Machine Setup and Configuration](vm_setup.md).
+- 	After done, you can continue to 1.2.
+
+## 1.2.	Install Kubernetes Tools to Cloud core and Edge core
```

Comment:
Install Kubernetes Tools to Cloud core and Edge core --> Install Kubernetes Tools

in 1.2 there doesn't seem to be anything installed about edge core and cloud core.

Also, for each PR comments from reviewers, comment "addressed" and wait for the commenter to resolve it.

Reply:
update and addressed